src/Authority/Authority.php

```diff
@@ -108,26 +108,38 @@ public function cannot($action, $resource, $resourceValue = null)
     /**
      * Define privilege for a given action and resource
      *
-     * @param string        $action Action for the rule
+     * @param array|string  $action Action for the rule
      * @param mixed         $resource Resource for the rule
      * @param Closure|null  $condition Optional condition for the rule
      * @return Rule
      */
     public function allow($action, $resource, $condition = null)
     {
+        if (is_array($action)) {
```

Comment:
Sorry, I thought I was waiting for a new commit!  We can't typehint `$action` to `array`, as that will break all backwards compatibility.  Doing something like `$action = [$action];` inside of the method body would do what I think you intend here.  Also, I'm not a huge fan of conditional returns one above another, plus I try to keep Authority PSR-1/2 compliant.  Still not sure how I feel about the return values, but I think the single / collection version is the only choice.